### DISCUSSION TITLE
PowerBI

### DISCUSSION THREAD

**Discussion by lzinga** [etsy/open-api#748] [Ideas]
I am new to using Etsy and have found the reporting to be quite lacking. I deal with data on a daily basis for my day to day job and work with powerbi quite a bit.  There may be a way to integrate it with powerbi but as of right now I have had an app request open for weeks with no responses.

So in hopes it gets accepted soon I was wondering if anyone has done this.

**Comment by etsyachristensen** [comment]
@lzinga You can email developers@etsy.com to ask for a status on your app. It may be a matter of your app being submitted just before we started re-approving apps certain apps. You'd have to make sure that your app description is filled out in full explaining the intent of the app and add your url.

**Reply by etsyachristensen** [reply]
I'm not a PowerBI user so I can't really speak to what's possible. I also don't know what data you are looking to collect so it's hard to say if the API will provide what you are looking for.

**Reply by lzinga** [reply]
I finally got access to the api so will be doing some testing to see if I can get it working within PowerBI. If you have any idea or help that could make the process easier it would be greatly appreciated.
